### PR TITLE
Syslog source IP test plan

### PR DESCRIPTION
**Pull request by JibinBao**
Add test plan for Syslog source IP test plan feature:

Syslog source IP HLD: https://github.com/JibinBao/sonic-mgmt/pull/new/syslog_test_plan

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@qiluo-msft  could you please help to review or suggest someone to do so?

**Review by qiluo-msft — APPROVED** [review]

**Comment by nazariig** [comment]
@liat-grozovik / @qiluo-msft shouldn't this PR go to the `sonic-mgmt` repo?

**Comment by liat-grozovik** [comment]
good question. @yxieca  what is the right place for test plans? sonic-mgmt repo or sonic?

File: doc/syslog/Syslog_Source_IP_test_plan.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
+# Syslog Source IP Test Plan
+
+## Related documents
+
+| **Document Name** | **Link** |
+|-------------------|----------|
+| Syslog Source IP HLD | [https://github.com/sonic-net/SONiC/blob/983fb02b909c236e06f2849ee990d4f3fd377516/doc/syslog/syslog-design.md]|
```
Comment (nazariig) [inline doc/syslog/Syslog_Source_IP_test_plan.md:7]: @JibinBao please either update with `https://github.com/sonic-net/SONiC/blob/master/doc/syslog/syslog-design.md` or remove

**Review by nazariig — CHANGES_REQUESTED** [review]

**Comment by yxieca** [comment]
> > good question. @yxieca what is the right place for test plans? sonic-mgmt repo or sonic?
> 
> @yxieca can you please clarify?

HLD goes here. Test plan goes to sonic-mgmt/docs. Cheers,

**Comment by prsunny** [comment]
> > > good question. @yxieca what is the right place for test plans? sonic-mgmt repo or sonic?
> > 
> > 
> > @yxieca can you please clarify?
> 
> HLD goes here. Test plan goes to sonic-mgmt/docs. Cheers,

File: doc/syslog/Syslog_Source_IP_test_plan.md (hunk 1 above)
Comment (JibinBao) [reply doc/syslog/Syslog_Source_IP_test_plan.md]: Update is done

**Comment by JibinBao** [comment]
Move it to  sonic-mgmt/docs
I have opened one new PR:https://github.com/Azure/sonic-mgmt/pull/5943

**Comment by JibinBao** [comment]
A new PR has already been opened

**Comment by JibinBao** [comment]
closed

**Pull request closed without merging**